dartagnan/src/main/java/com/dat3m/dartagnan/encoding/PropertyEncoder.java

```diff
@@ -249,12 +239,18 @@ public BooleanFormula encodeDataRaces() {
                 if(t1 == t2) {
                     continue;
                 }
-                for(Event e1 : t1.getCache().getEvents(FilterMinus.get(FilterBasic.get(WRITE), FilterBasic.get(INIT)))) {
+                for (Event e1 : t1.getEvents()) {
+                    if (!e1.hasFilter(Tag.WRITE) || e1.hasFilter(Tag.INIT)) {
+                        continue;
+                    }
                     MemEvent w = (MemEvent)e1;
                     if (!w.canRace()) {
                         continue;
                     }
-                    for(Event e2 : t2.getCache().getEvents(FilterMinus.get(FilterBasic.get(Tag.MEMORY), FilterBasic.get(INIT)))) {
+                    for(Event e2 : t2.getEvents()) {
+                        if (!e2.hasFilter(Tag.WRITE) || e2.hasFilter(Tag.INIT)) {
```

Comment:
This was a check for `Memory \ Init` before, which seems to have changed to `Write \ Init`.